src/dmd/aggregate.d

```diff
@@ -57,6 +57,28 @@ enum ClassKind : ubyte
     c,
 }
 
+/**
+ * Give a nice string for a class kind for error messages
+ * Params:
+ *     c = class kind
+ * Returns:
+ *     0-terminated string for `c`
+ */
+const(char)* toChars(ClassKind c)
```

Comment:
It seems that we already have a similar function that works on a linkage, not no a ClassKind: https://github.com/dlang/dmd/blob/master/src/dmd/hdrgen.d#L2967 

We most likely don't need both, so I suggest finding a way to merge the 2 concepts.

Reply:
I don't think it's this PR's responsibility to merge the `ClassKind` and `LINK` concepts, if that's even a good idea in the first place. (I only use `extern(C)` and `extern(D)` myself so I'm not sure)